src/ansi-c/ansi_c_internal_additions.cpp

```diff
@@ -216,6 +216,21 @@ void ansi_c_internal_additions(std::string &code)
     // This function needs to be declared, or otherwise can't be called
     // by the entry-point construction.
     "void " INITIALIZE_FUNCTION "(void);\n"
+    "\n"
+    // frame specifications for contracts
+    // Type that describes assignable memory locations
+    "typedef void " CPROVER_PREFIX "assignable_t;\n"
```

Comment:
Why is this typedef required, couldn't we just use `void` in all the places that use it?